3.6/oasis/backups.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+description: Quick guide about backups in Oasis for ArangoDB.
+title: Backups in Oasis
+---
+
+# How to create backups
+
+To backup data in Oasis for an ArangoDB installation, navigate to the backup section of your deployment created previously.
+
+![Backup ArangoDB](../images/oasis-backup-section.png)
+
+There are two types of backups. Periodic backups taken through backup policies and manual backups taken by hand.
```

Comment:
There are two ways to create backups.
Create periodic backups using a <b>Backup policy</b>, or create a backup manually.

(if is not really a type)